Failing test: Security Solution Cypress.x-pack/test/security_solution_cypress/cypress/e2e/detection_response/rule_management/prebuilt_rules/install_workflow·cy·ts - Detection rules, Prebuilt Rules Installation and Update workflow Installation of prebuilt rules "before each" hook for "should install prebuilt rules one by one" "before each" hook for "should install prebuilt rules one by one"

A test failed on a tracked branch

```
AssertionError: Timed out retrying after 150000ms: Expected to find element: `[data-test-subj="addElasticRulesButton"]`, but never found it.

Because this error occurred during a `before each` hook we are skipping the remaining tests in the current suite: `Installation of prebuilt rules`
    at clickAddElasticRulesButton (webpack:///./tasks/prebuilt_rules.ts:22:5)
    at Context.eval (webpack:///./e2e/detection_response/rule_management/prebuilt_rules/install_workflow.cy.ts:55:35)
```

First failure: [CI Build - main](https://buildkite.com/elastic/kibana-on-merge/builds/44699#018f3a42-ca28-4070-af92-70ad32cec215)

<!-- kibanaCiData = {"failed-test":{"test.class":"Security Solution Cypress.x-pack/test/security_solution_cypress/cypress/e2e/detection_response/rule_management/prebuilt_rules/install_workflow·cy·ts","test.name":"Detection rules, Prebuilt Rules Installation and Update workflow Installation of prebuilt rules \"before each\" hook for \"should install prebuilt rules one by one\" \"before each\" hook for \"should install prebuilt rules one by one\"","test.failCount":2}} -->

--- a/x-pack/solutions/security/test/security_solution_cypress/cypress/e2e/detection_response/rule_management/prebuilt_rules/customization/revert_prebuilt_rule.cy.ts
+++ b/x-pack/solutions/security/test/security_solution_cypress/cypress/e2e/detection_response/rule_management/prebuilt_rules/customization/revert_prebuilt_rule.cy.ts
@@ -37,6 +37,7 @@ import { createRule, patchRule } from '../../../../../tasks/api_calls/rules';
 import { login } from '../../../../../tasks/login';
 
 import { visitRulesManagementTable } from '../../../../../tasks/rules_management';
+
 describe(
   'Detection rules, Prebuilt Rules reversion workflow',
   {

--- a/x-pack/solutions/security/test/security_solution_cypress/cypress/e2e/detection_response/rule_management/prebuilt_rules/installation/install_error_handling.cy.ts
+++ b/x-pack/solutions/security/test/security_solution_cypress/cypress/e2e/detection_response/rule_management/prebuilt_rules/installation/install_error_handling.cy.ts
@@ -45,7 +45,6 @@ import {
 } from '../../../../../tasks/prebuilt_rules';
 import { visitRulesManagementTable } from '../../../../../tasks/rules_management';
 
-// https://github.com/elastic/kibana/issues/179970
 describe(
   'Detection rules, Prebuilt Rules Installation - Error handling',
   { tags: ['@ess', '@serverless', '@skipInServerlessMKI'] },

--- a/x-pack/solutions/security/test/security_solution_cypress/cypress/e2e/detection_response/rule_management/prebuilt_rules/installation/install_notifications.cy.ts
+++ b/x-pack/solutions/security/test/security_solution_cypress/cypress/e2e/detection_response/rule_management/prebuilt_rules/installation/install_notifications.cy.ts
@@ -50,7 +50,6 @@ describe(
         cy.get(ADD_ELASTIC_RULES_BTN).should('have.text', 'Add Elastic rules');
       });
 
-      // https://github.com/elastic/kibana/issues/179967
       it(
         'does NOT display install notifications when latest rules are installed',
         { tags: ['@skipInServerlessMKI'] },
@@ -72,7 +71,6 @@ describe(
       );
     });
 
-    // https://github.com/elastic/kibana/issues/179968
     describe('Notifications', () => {
       const PREBUILT_RULE = createRuleAssetSavedObject({
         name: 'Test prebuilt rule 1',

--- a/x-pack/solutions/security/test/security_solution_cypress/cypress/e2e/detection_response/rule_management/prebuilt_rules/installation/install_update_authorization.cy.ts
+++ b/x-pack/solutions/security/test/security_solution_cypress/cypress/e2e/detection_response/rule_management/prebuilt_rules/installation/install_update_authorization.cy.ts
@@ -66,11 +66,7 @@ const loginPageAsWriteAuthorizedUser = (url: string) => {
   visit(url);
 };
 
-// https://github.com/elastic/kibana/issues/179965
-// Failing: See https://github.com/elastic/kibana/issues/182485
-// Failing: See https://github.com/elastic/kibana/issues/182483
-// Failing: See https://github.com/elastic/kibana/issues/182486
-describe.skip(
+describe(
   'Detection rules, Prebuilt Rules Installation and Update - Authorization/RBAC',
   { tags: ['@ess', '@serverless', '@skipInServerlessMKI'] },
   () => {

--- a/x-pack/solutions/security/test/security_solution_cypress/cypress/e2e/detection_response/rule_management/prebuilt_rules/installation/install_update_authorization.cy.ts
+++ b/x-pack/solutions/security/test/security_solution_cypress/cypress/e2e/detection_response/rule_management/prebuilt_rules/installation/install_update_authorization.cy.ts
@@ -12,7 +12,12 @@ import {
 } from '@kbn/security-solution-plugin/common/constants';
 import { ROLES } from '@kbn/security-solution-plugin/common/test';
 
+import { resetRulesTableState } from '../../../../../tasks/common';
 import { createRuleAssetSavedObject } from '../../../../../helpers/rules';
+import {
+  deleteAlertsAndRules,
+  deletePrebuiltRulesAssets,
+} from '../../../../../tasks/api_calls/common';
 import {
   createAndInstallMockedPrebuiltRules,
   installMockPrebuiltRulesPackage,
@@ -72,10 +77,13 @@ describe(
   () => {
     before(() => {
       installMockPrebuiltRulesPackage();
+      preventPrebuiltRulesPackageInstallation();
     });
 
     beforeEach(() => {
-      preventPrebuiltRulesPackageInstallation();
+      deletePrebuiltRulesAssets();
+      deleteAlertsAndRules();
+      resetRulesTableState();
     });
 
     describe('User with read privileges on Security Solution', () => {
@@ -113,15 +121,15 @@ describe(
         cy.get(RULES_UPDATES_TAB).should('not.exist');
 
         // Navigate to Rule Update tab anyways via URL
-        // and assert that rules cannot be selected and all
-        // upgrade buttons are disabled
         cy.visit(`${APP_PATH}${RULES_UPDATES}`);
-        cy.get(UPGRADE_ALL_RULES_BUTTON).should('be.disabled');
 
-        // Upgrade button and selection checkbox should not be visible
+        // Check that upgrade buttons are not visible
+        cy.get(UPGRADE_ALL_RULES_BUTTON).should('not.exist');
         cy.get(getUpgradeSingleRuleButtonByRuleId(OUTDATED_RULE_1['security-rule'].rule_id)).should(
           'not.exist'
         );
+
+        // Check that rule selection checkbox is not visible
         cy.get(RULE_CHECKBOX).should('not.exist');
       });
     });

--- a/x-pack/solutions/security/test/security_solution_cypress/cypress/e2e/detection_response/rule_management/prebuilt_rules/installation/install_via_fleet.cy.ts
+++ b/x-pack/solutions/security/test/security_solution_cypress/cypress/e2e/detection_response/rule_management/prebuilt_rules/installation/install_via_fleet.cy.ts
@@ -20,8 +20,7 @@ import { expectManagementTableRules } from '../../../../../tasks/alerts_detectio
 
 const PREBUILT_RULES_PACKAGE_INSTALLATION_TIMEOUT_MS = 120000; // 2 minutes
 
-// Failing: See https://github.com/elastic/kibana/issues/228945
-describe.skip(
+describe(
   'Detection rules, Prebuilt Rules Installation Workflow (Real package)',
   { tags: ['@ess', '@serverless'] },
   () => {

--- a/x-pack/solutions/security/test/security_solution_cypress/cypress/e2e/detection_response/rule_management/prebuilt_rules/management/management.cy.ts
+++ b/x-pack/solutions/security/test/security_solution_cypress/cypress/e2e/detection_response/rule_management/prebuilt_rules/management/management.cy.ts
@@ -49,7 +49,6 @@ const PREBUILT_RULE_ASSETS = Array.from(Array(5)).map((_, i) =>
   })
 );
 
-// https://github.com/elastic/kibana/issues/179973
 describe('Prebuilt rules', { tags: ['@ess', '@serverless', '@skipInServerlessMKI'] }, () => {
   before(() => {
     installMockPrebuiltRulesPackage();

